### PR TITLE
Event hackathon

### PR DESCRIPTION
**Pull request by christopher-hakkaart**
- Add Barcelona Hackathon
- Add Boston Hackathon
- Add banner now to help promotion

### CODE REVIEW

**Comment by christopher-hakkaart** [comment]
@mashehu - are there any blockers on this? Preview looks good!
@mribeirodantas - can you give it a review, please 🙏

**Comment by mashehu** [comment]
the announcement banner disappears on firefox, looking into it atm

File: src/content/events/2023/hackathon-november-2023.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Hackathon - November 2023 (Boston)
+subtitle: An in-person hackathon held in Boston
+type: hackathon
+start_announcement: '2023-10-21 12:00 CEST'
+start_date: '2023-11-28'
+start_time: '10:00 CEST'
```
Comment (mashehu) [inline src/content/events/2023/hackathon-november-2023.md:7]: Are you sure these times are CEST?

Comment (christopher-hakkaart) [reply src/content/events/2023/hackathon-november-2023.md:7]: Good spot! Should be EST

**Review by mribeirodantas — APPROVED** [review]

**Comment by mashehu** [comment]
oops sorry, that last push was to the wrong branch.

**Comment by mashehu** [comment]
Merge conflict is fixed. @christopher-hakkaart this PR should be good to go know

**Pull request merged**